core/src/main/kotlin/org/evomaster/core/output/service/TestSuiteWriter.kt

```diff
@@ -472,7 +472,17 @@ class TestSuiteWriter {
             addUsing("EvoMaster.Controller", lines)
         }
 
-        lines.addEmpty(4)
+        if (format.isPython()) {
+            lines.add("import json")
+            lines.add("import unittest")
+            lines.add("import requests")
+            createRequirementsTxt("requests==2.25.1")
```

Comment:
should not create an external txt file. as for Java and Koltin, it is up to user to set dependencies on (eg, see [https://github.com/EMResearch/EvoMaster/blob/master/docs/library_dependencies.md](https://github.com/EMResearch/EvoMaster/blob/master/docs/library_dependencies.md)). 
if really need to specify something here, you can add it as a comment

Reply:
Oh, I see. Hadn't checked that library_dependencies article. Will remove this now. Regarding if need documentation for this case, requests is not included in the python standard library, but it's a widely used library, so anyone creating python tests will most likely already have the `requests` library in their python env

Reply:
Done